ace-indexer/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>curated-training-all</artifactId>
+        <groupId>edu.isi.vista</groupId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>ace-indexer</artifactId>
+    <dependencies>
```

Comment:
Recommendation: use the `appassembler` plugin to create an executable script. This way, after building the project, one can easily run the indexer through the command line with something like
`ace-indexer/target/appassembler/bin/indexAce path/to/file.params`.

See https://github.com/isi-vista/curated-training-annotator/blob/master/gigaword-indexer/pom.xml for reference.